src/annotator/test/annotation-sync-test.js

```diff
@@ -169,4 +165,41 @@ describe('AnnotationSync', () => {
       assert.calledWith(fakeBridge.call, 'sync', [{ msg: ann, tag: ann.$tag }]);
     });
   });
+
+  describe('#destroy', () => {
+    it('respond with errors to events from the sidebar', () => {
```

Comment:
As noted above, I think causing the Bridge to be destroyed in this case is unexpected. Logically what we want to happen when `AnnotationSync` is destroyed is to disconnect the bridge. The simplest way to implement that currently is to make the event handlers do nothing.